webapp/store/app/utils/index.js

```diff
@@ -0,0 +1,16 @@
+import { Area } from '@common/country'
+
+export const sortCountries = (countries, i18n) => {
+  const compareListName = Area.getCompareListName(i18n)
+  const compareCountries = (country1, country2) => compareListName(country1.countryIso, country2.countryIso)
+  return [...countries].sort(compareCountries)
+}
+
+export const sortRegions = (regions, i18n) => {
+  const compareListName = Area.getCompareListName(i18n)
+  return [...regions].sort((r1, r2) => compareListName(r1.regionCode, r2.regionCode))
+}
+
+export const sortRegionGroups = (regionGroups) => {
+  return [...regionGroups].sort((rg1, rg2) => rg1.order > rg2.order)
+}
```

Comment:
how is going to grow this file?
maybe it is a good idea starting at this point split the utils folder by domain, ( on folder or file per regions other for countries, ) but maybe I need more context about the domain and the amount of use cases.

Reply:
I feel scare about TS when we like to implement this functions as 
`export const sortCountries = ({countries, i18n}) => {`
vs
`export const sortCountries = (countries, i18n) => {`

doing the params in this way is easier to add new params and at the end more scalable and easier to mock

Reply:
I needed to create this file - hopefully for now - because of circular import issue with eslint.
Regarding the second comment:
I think prototyping with TypeScript is slower - but after prototyping phase the project is easier to upkeep